pymc3/distributions/simulator.py

```diff
@@ -15,65 +15,64 @@
 import logging
 import numpy as np
 from .distribution import NoDistribution, draw_values
+from scipy.spatial import cKDTree
 
 __all__ = ["Simulator"]
 
 _log = logging.getLogger("pymc3")
 
 
 class Simulator(NoDistribution):
+    r"""
+    Define a simulator, from a Python function, to be used in ABC methods.
+
+    function: function
```

Comment:
To be consistent with the rest of the code base:
```
    Parameters
    ----------
    function: callable
    ...
```

Reply:
in case it is not obvious - i am suggesting to also add
```python
    Parameters
    ----------
```

Reply:
(－‸ლ)